geomesa-datastore-bundle/geomesa-datastore-processors/src/main/resources/META-INF/services/org.apache.nifi.processor.Processor

```diff
@@ -1,4 +1,5 @@
 org.geomesa.nifi.datastore.processor.ConvertToGeoAvro
+org.geomesa.nifi.datastore.processor.ConvertToGeoFile
```

Comment:
Is this a new NiFi Processor?  If so, should there be a docs PR?

Reply:
yes, i need to update the docs in main gm, thanks for the reminder